testsuit/funcExecutor.go

```diff
@@ -38,7 +37,7 @@ func executeFunc(fn reflect.Value, args ...interface{}) (res *m.ProtoExecutionRe
 			res.ScreenShot = getScreenshot()
 			res.Failed = true
 			res.ExecutionTime = time.Since(start).Nanoseconds() / int64(time.Millisecond)
-			res.StackTrace = strings.SplitN(string(debug.Stack()), "\n", 9)[8]
+			res.StackTrace = caller(4)
```

Comment:
It will be good to get the trace when it's on gauge debug mode or a print it using a logger.debug